packages/create-discord-bot/template/TypeScript/tsconfig.json

```diff
@@ -8,6 +8,8 @@
 		"moduleResolution": "NodeNext",
 		"target": "ESNext",
 		"outDir": "dist",
-		"skipLibCheck": true
+		"skipLibCheck": true,
+		"strictNullChecks": true,
+		"allowImportingTsExtensions": true
```

Comment:
This should only be enabled for deno/bun, not every TS project! 👀

Reply:
Make sense